{{ cookiecutter.__package_name_kebab_case }}/README.md

```diff
@@ -78,6 +81,7 @@ To serve this app, run `docker compose up app` and open [localhost:8000](http://
             {%- endif %}
             EOF
             ```
+
     {%- if cookiecutter.private_package_repository_name %}
```

Comment:
Similar to the previous comment:

This newline will be removed by the `{%- if cookiecutter.private_package_repository_name %}` line, so we instead need to insert it below the if:
```suggestion
    {%- if cookiecutter.private_package_repository_name %}

```

We also need to a newline on line 92 in case the if statement is not true.